### ISSUE TITLE
UPN not working

### ISSUE THREAD

**Issue by grakist** [multiOTP/multiotp#81]
On the windows credentials provider; when I enable "Send the username to multiOTP in UPN format"; it doesn't send any username anymore.

When I enable this setting the <UserID> tag is empty in the request to the server. With this setting disabled everything works as expected.
 
I tried several versions including the last. They all exhibit this problem.

BR, Eelco

**Comment by syscoch** [comment]
Hello,
When you do your tests, is your computer connected to the LAN ?
Is the computer part of a Windows domain ?

Have a good day

**Comment by grakist** [comment]
Hi,
Yes the PC is connected to a LAN network. I tried it on different machines on different networks. Some PC's are standalone and others are member of a Windows Domain. Alle the PC's run Windows 10 Pro (1909). The OTP server runs on Linux.

I did a https debug on the OTP server and I see that when this UPN setting is enable no username at all is send. There is communication; but the username is empty. No username, no at-sign and no domain name. So the problem must be at the windows client and not the OTP server.

The moment I disable this feature the username is send as usual.

I use one OTP server for multiple PC's. Some of the PC's have users with the same name. That's why it would be very convenient to add the computer name (or domain name) to the OTP username.
